cli/fabric.c

```diff
@@ -729,12 +749,12 @@ static int hvd_detail_body_print(struct switchtec_dev *dev,
 	char bdf_str1[32];
 	char bdf_str2[32];
 	char vep_type_str[32];
-	uint64_t enable_bitmap;
-	uint64_t bitmap;
+	unsigned long long enable_bitmap;
```

Comment:
Why were these changed from uint64_t? Isn't uint64_t preferable since it's a guaranteed 64 bits?

Reply:
`uint64_t` is fine for data type, but when we try to print on 64-bit Linux with "%llx", it expects 'unsigned long long', but `uint64_t` is defined as 'unsigned long', so we will get a mismatch warning.

On 32-bit system, uint64_t is defined as 'unsigned long long' and we won't have issue. 

So the best fix across these different architectures is to define the data type as 'unsigned long long', and print with "%llx", so the consistency is maintained.

Reply:
If you use the PRI* macros on Linux as well, I think you could keep it at uint64_t then just use PRIx64. This should be compatible across platforms.

Reply:
Updated based on your suggestion. Thanks.